lib/src/evaluation_context.dart

```diff
@@ -25,6 +26,16 @@ abstract interface class EvaluationContext {
     }
   }
 
+  /// The current evaluation context, or null if there isn't a Sass stylesheet
+  /// currently being evaluated.
+  static EvaluationContext? get currentOrNull {
```

Comment:
I think this could be private.

Reply:
Addressed in https://github.com/sass/dart-sass/pull/2117/commits/63c17e31c3f5f6b168bba3f32ecc33ac65113593